### PR TITLE
refractor(patient invoice)set focus on quantity after selecting the code

### LINKED ISSUE #2837: Automatically go to "quantity" after "code" on Patient Invoice
The Patient Invoice module is hard to use (compared to 1.x) because after selecting the code in the grid, everything blurs.  An ideal way to improve this would be to have application select "quantity" after entering the "code" automatically.  This would allow a user to rapidly progress through the module.

### PR DESCRIPTION
**Pull request by jeremielodi**
closes #2837

### CODE REVIEW

File: client/src/modules/invoices/patientInvoiceForm.service.js [hunk 1]
```diff
@@ -433,7 +433,10 @@ function PatientInvoiceFormService(
     if (angular.isDefined(price)) {
       item.applyPriceList(price);
     }
-
+    // set focus on quantity field
+    setTimeout(() => {
+      document.getElementById(item.uuid).focus();
+    }, 100);
```
Comment (mbayopanda) [inline client/src/modules/invoices/patientInvoiceForm.service.js]: Could you use the angular version `$timeout`.

**Review by mbayopanda — COMMENTED** [review]
LGTM

There is a comment on an Angularjs wrapper for setTimeout, which is more natural for an Angular application.

**Review by jniles — COMMENTED** [review]
Code looks okay to me.  The change from `setTimeout` -> `$timeout` was a good one.  You might consider doing the same thing with `document` -> `$document`.

I'm curious how you plan to write a test for this?

**Comment by jniles** [comment]
The behavior here works well.  👍 

I'll try to write a test for this.

**Comment by jniles** [comment]
Alright, test added.  @mbayopanda can you give us a review?

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

bors r+

**Pull request merged**